examples/browser/package.json

```diff
@@ -10,7 +10,7 @@
     "monaco-languages": "^0.7.0",
     "monaco-typescript": "^2.1.2",
     "reflect-metadata": "^0.1.10",
-    "theia": "file:../../../Theia"
+    "theia": "file:../../../theia"
```

Comment:
Well, there could be legit reasons in the future to have the repo checked out in a directory named something else than "theia", like if you need multiple versions checked out at the same time, but hardcoding the path here would prevent that.  If we can easily avoid putting this restriction, why not?